.env.alfajores

```diff
@@ -78,9 +78,21 @@ EKSPORTISTO_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/eksportisto"
 EKSPORTISTO_DOCKER_IMAGE_TAG="c88f7e8babcdb01062a6a12ae729705ba9769add"
 EKSPORTISTO_SUFFIX='1'
 
+# Attestation Bot variables
+ATTESTATION_BOT_INITIAL_WAIT_SECONDS=1200
+ATTESTATION_BOT_IN_BETWEEN_WAIT_SECONDS=1800
+ATTESTATION_BOT_MAX_ATTESTATIONS=32
+
+# If this is set, the bot will skip ODIS and use this salt (needed for baklava, where ODIS is not deployed)
+ATTESTATION_BOT_SKIP_ODIS_SALT=bot
+
 ATTESTATION_SERVICE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/celo-monorepo"
 ATTESTATION_SERVICE_DOCKER_IMAGE_TAG="attestation-service-v1.2.0"
 
+TELEKOM_FROM="+14157389085"
+SMS_PROVIDERS=twilio,nexmo,telekom
```

Comment:
could also add MessageBird here

Reply:
(or else just rely on telekom?)

Reply:
Prefer to get this in as its been open for a while and add message bird in a separate PR